.github/workflows/test_and_lint.yml

```diff
@@ -0,0 +1,65 @@
+name: build
+
+on:
+  push:
+    branches: [ github-actions ]
+  pull_request:
+    branches: [ github-actions ]
+
+jobs:
+
+  build-and-test:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        node-version: [10.x]
+
+    steps:
+    - name: install dependencies on ubuntu
+      if: startsWith(matrix.os,'ubuntu')
+      run: |
+        sudo apt install -y make gcc pkg-config build-essential libx11-dev libxkbfile-dev 
+    
+    - uses: actions/checkout@v2
+    
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Get yarn cache directory path
+      id: yarn-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    
+    - name: Cache node_modules with yarn
+      uses: actions/cache@v2
+      id: yarn-cache # use this to check for `cache-hit` (`steps.yarn-cache.outputs.cache-hit != 'true'`)
+      with:
+        path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-yarn-
+    
+    - run: yarn --frozen-lockfile
+    
+    - run: yarn test
```

Comment:
Add the test in the packages.json, like you did in your previous abandoned PR, just echo something. Then if the build passes, you can change the 'on' branches above and this is ready to go in!